src/compiler/parser.ts

```diff
@@ -6820,6 +6820,7 @@ namespace ts {
                     typedefTag.comment = parseTagComments(indent);
 
                     typedefTag.typeExpression = typeExpression;
+                    let end: number;

```

Comment:
does this quirk apply to callback tags too?

Reply:
From our discussion: yes, and nested `@param` types too, but the problem isn't bad enough to fix right now. Instead we should fix it by fixing the jsdoc scanner to ignore whitespace (and change the way we generate tag descriptions).